monitors.yml

```diff
@@ -195,3 +195,18 @@ monitors:
         warning: 400000
     query: avg:zuul.pipeline.check_github.job.bonnyci_run_check.wait_time.max{host:zuul} > 800000
     type: metric alert
+  - message: |
+      {{#is_warning}}Only two nodes available{{/is_warning}} 
```

Comment:
I also don't love that these messages are so tied to their threshold values. You can template the messages, I believe, to give specific counts, that way if we tweak the thresholds, the messages don't need to be tweaked.